docker/cstor.Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 The OpenEBS Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:18.04 as build
+
+RUN mkdir -p libcstor
+COPY . libcstor/
+
+# Final
+FROM openebs/cstor-base:ci
+
+COPY --from=build libcstor/docker/entrypoint-poolimage.sh /usr/local/bin/
```

Comment:
I would move the `COPY` statement below the ARGs and LABEL as those layers are cached and don't change frequently.